### PR TITLE
id_update

### PR DESCRIPTION
**Pull request by ejaronne**
update_ids

### CODE REVIEW

File: controls/SV-204393.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@
   tag gtitle: 'SRG-OS-000023-GPOS-00006'
   tag satisfies: ['SRG-OS-000023-GPOS-00006', 'SRG-OS-000024-GPOS-00007', 'SRG-OS-000228-GPOS-00088']
   tag gid: 'V-204393'
-  tag rid: 'SV-204393r603261_rule'
+  tag rid: 'AMZL-02-710030r603261_rule'
```
Comment (karikarshivani) [inline controls/SV-204393.rb:50]: Tags to remove:
- legacy
- gid
- rid
- fix_id

File: controls/SV-204393.rb [hunk 2]
```diff
@@ -47,7 +47,7 @@
...
   tag stig_id: 'RHEL-07-010030'
```
Comment (karikarshivani) [inline controls/SV-204393.rb:51]: Update `stig_id` to the control id

File: controls/SV-204393.rb (hunk 1 above)
Comment (karikarshivani) [reply controls/SV-204393.rb:50]: @raziz086

File: controls/SV-204393.rb (hunk 2 above)
Comment (karikarshivani) [reply controls/SV-204393.rb:51]: @damif5

Comment (raziz086) [reply controls/SV-204393.rb]: Reviewed updates and it looks good.

**Review by ejaronne — COMMENTED** [review]
LGTM

**Pull request merged**